src/org/usfirst/frc/team467/robot/RobotMap.java

```diff
@@ -63,21 +63,23 @@ public static void init(RobotID id) {
 	public static double WHEEL_BASE_WIDTH;
 	public static double CamToCenterWidthInches;
 	public static double CamToCenterLengthInches;	
-
+	
+    public static double maxAcceleration = 0.1;
+	
 	/**
 	 * Used to ensure that all Talon SRX outputs are relative to a fixed value.
 	 * If the available voltage is below the nominal and a value about that is
 	 * requested, the output will be 100%.
 	 */
 	public static final double NOMINAL_BATTERY_VOLTAGE = 12.0;
 
-	// The circumference of the wheels for use in determining distance in
+	// The circumference of the wheels for use in determining distance in inches in 
 	// position mode
-	public static final double WHEELPOD_CIRCUMFERENCE = 19.74; //20.5139; //21.43
+	public static final double WHEEL_CIRCUMFERENCE = 6*Math.PI;
 
 	// The number of encoder ticks per one revolution of the wheel. This is used
 	// for correctly determining RPM and position.
-	public static final int WHEELPOD_ENCODER_CODES_PER_REVOLUTION = 256;
+	public static final int WHEEL_ENCODER_CODES_PER_REVOLUTION = 256;
```

Comment:
We should remember to fix this in a later branch. Should have been done in motion magic branch actually.